src/main/resources/avro/sequenceAnnotations.avdl

```diff
@@ -0,0 +1,145 @@
+@namespace("org.ga4gh.models")
+/**
+This protocol defines annotations on GA4GH genomic sequences It includes two
+types of annotations: continuous and discrete hierarchical.
+
+The discrete hierarchical annotations are derived from the Sequence Ontology
+(SO) and GFF3 work 
+
+   http://www.sequenceontology.org/gff3.shtml
+
+The goal is to be able to store annotations using the GFF3 and SO conceptual
+model, although there is not necessarly a one-to-one mapping in Avro records
+to GFF3 records.
+
+The minimum requirement is to be able to accurately represent the current
+state of the art annotation data and the full SO model.  Feature is the
+core generic record which corresponds to the a GFF3 record.
+*/
+protocol SequenceAnnotations {
+
+  import idl "common.avdl";
+
+  /**
+  Type defining a collection of attributes associated with various protocol
+  records.  Each attribute is a name that maps to an array of one or more
+  values.  To work around Avro restrictions, three fields are define, one for
+  each type of value.  However attribute names must be unique across all three
+  fields.  If possible, multi-part string values should be split into array
+  elements instead of adding separator syntax that need to parsed.
+  */
+  record Attributes {
```

Comment:
Since this is defining `org.ga4gh.models.Attributes`, shouldn't it be placed in `common.avdl` and used to replace all the `map<array<string>>` catch-all fields over in `variants.avdl`?

Or should this object maybe be `SequenceAnnotationAttributes` or something else more specific?